packages/flutter/lib/src/material/divider.dart

```diff
@@ -31,17 +32,18 @@ class Divider extends StatelessWidget {
     Key key,
     this.height = 16.0,
     this.indent = 0.0,
+    this.axis = Axis.horizontal,
     this.color
-  }) : assert(height >= 0.0),
-       super(key: key);
+  }) : assert(height >= 0.0 && axis != null),
+        super(key: key);
 
-  /// The divider's vertical extent.
+  /// The divider's dimensional extent.
   ///
-  /// The divider itself is always drawn as one device pixel thick horizontal
-  /// line that is centered within the height specified by this value.
+  /// The divider itself is always drawn as one device pixel thick
+  /// line that is centered within the height or width specified by this value.
   ///
-  /// A divider with a height of 0.0 is always drawn as a line with a height of
-  /// exactly one device pixel, without any padding around it.
+  /// A divider with a [width] of 0.0 is always drawn as a line with a width or
+  /// height of exactly one device pixel, without any padding around it.
```

Comment:
This is a bit unfortunate, but what can you do...

Reply:
yah...